source/features/remove-projects-sidebar.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+The `Projects` sidebar is hidden from repositories and profiles when there are no projects.
+The sidebar can't be used to create new projects.
+New projects can still be created via the [`Create new…` menu](https://user-images.githubusercontent.com/1402241/34909214-18b6fb2e-f8cf-11e7-8556-bed748596d3b.png).
```

Comment:
Unrelated. The first line is enough

Reply:
I just rewrote the feature to get the results without flicker.

Reply:
It's not linted, but I'm trying to make point of not putting on pretenses. If this still doesn't agree with you I'd rather be compensated all the same, and avoid making any PRs in future.

Reply:
It's good to see that

Reply:
> I'd rather be compensated all the same

For the record, bounties are just rewards, not compensation. PRs with bounties can get rejected just like any other.

Reply:
> It's not linted, but I'm trying to make point of not putting on pretenses.

What does this mean?

Reply:
> PRs with bounties can get rejected just like any other.

They are not exactly like any other PR in that we expect the PR submitter to put in extra effort in their work.